ultravox/model/ultravox_model.py

```diff
@@ -16,10 +16,7 @@
 from .whisper_model_modified import WhisperEncoder as ModifiedWhisperEncoder
 
 
-class UltravoxModel(
-    transformers.LlamaPreTrainedModel,
-    transformers.GenerationMixin,
```

Comment:
Is transformers.GenerationMixin no longer needed?

Reply:
It's apparently part of `PretrainedModel` (parent of `LlamaPretraiendModel`). Not sure if this was always the case or just a new change. That means it was being added twice for no reason.